platform/ios/src/MGLMapView.mm

```diff
@@ -1567,7 +1571,9 @@ - (void)handleCalloutAccessoryTapGesture:(UITapGestureRecognizer *)tap
     if ([self.delegate respondsToSelector:@selector(mapView:annotation:calloutAccessoryControlTapped:)])
     {
         NSAssert([tap.view isKindOfClass:[UIControl class]], @"Tapped view %@ is not a UIControl", tap.view);
-        [self.delegate mapView:self annotation:self.selectedAnnotation
+        id <MGLAnnotation> selectedAnnotation = self.selectedAnnotation;
+        NSAssert(selectedAnnotation != NULL, @"Selected annotation should not be NULL.");
```

Comment:
Use `nil` instead of `NULL`. `id <MGLAnnotation>` is an Objective-C type, not a C type.